.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
I am just curious, what this is supposing to do?

Reply:
It's added by husky. When doing `husky install`, it creates `.husky/` with that `.gitignore` and a `_/` with a .sh in it.